app/controllers/insights_cloud/hits_controller.rb

```diff
@@ -23,9 +23,9 @@ def show
 
     def resolutions
       if remediation_all_selected_param
-        hits = InsightsHit.with_playbook.search_for(params[:query])
+        hits = resource_base.with_playbook.search_for(params[:query])
```

Comment:
How about using `resource_base_with_search`?

Reply:
`_with_search` hard codes `:search` parameter instead of `:query`.